plugins/conditions/cll/templates/forms/cll_regimen_form.html

```diff
@@ -10,7 +10,7 @@
   </div>
 </div>
 {% custom_datepicker field="CLLRegimen.start_date" no_future=False change="$root.patientValidator.regimen_start(editing.cll_regimen.start_date, editing.cll_regimen, the_episode); $root.patientValidator.regimen_end(editing.cll_regimen.end_date, editing.cll_regimen, the_episode);" date_after="$root.patientValidator.entrytool_helper.getDiagnosis($root.patientValidator.patient).diag_date" date_after_diff="0" date_after_message=_("Start Date must be after the patient date of diagnosis") validator="regimen_start" required=True %}
-{% custom_datepicker field="CLLRegimen.end_date" no_future=False change="$root.patientValidator.regimen_start(editing.cll_regimen.start_date, editing.cll_regimen, the_episode); $root.patientValidator.regimen_end(editing.cll_regimen.end_date, editing.cll_regimen, the_episode);" date_after="editing.regimen.start_date" date_after_diff="0" date_after_message=_("End Date must be after the start date") validator="regimen_end" %}
+{% custom_datepicker field="CLLRegimen.end_date" no_future=False change="$root.patientValidator.regimen_start(editing.cll_regimen.start_date, editing.cll_regimen, the_episode); $root.patientValidator.regimen_end(editing.cll_regimen.end_date, editing.cll_regimen, the_episode);" date_after="editing.regimen.start_date" date_after_diff="0" date_after_message=_("End Date must be after the start date") validator="regimen_end" ngrequired='$root.patientValidator.validateOnlyOneOpenRegimen(editing.cll_regimen.end_date, editing.cll_regimen, the_episode)' ngrequired_error="There can only be one open regimen at a time" %}
```

Comment:
Error message should be translated as well

Reply:
It's translated in the template on the other side
https://github.com/solventrix/HONEUR_eCRF/pull/127/files#diff-16923d18c018383f03e43953f055fe5d89b9563c456967d9cfb47cd4483e22b4R73

I put it there because that line of code is already too long I think.